datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/__tests__/filterSchemaRows.test.ts

```diff
@@ -35,6 +39,80 @@ describe('filterSchemaRows', () => {
         expect(expandedRowsFromFilter).toMatchObject(new Set());
     });
 
+    it('should properly filter schema rows based on description', () => {
+        const filterText = 'testing description';
+        const editableSchemaMetadata = { editableSchemaFieldInfo: [] };
+        const { filteredRows, expandedRowsFromFilter } = filterSchemaRows(
+            rows,
+            editableSchemaMetadata,
+            filterText,
+            testEntityRegistry,
+        );
+
+        expect(filteredRows).toMatchObject([{ fieldPath: 'testing' }]);
+        expect(expandedRowsFromFilter).toMatchObject(new Set());
+    });
+
+    it('should properly filter schema rows based on description regardless of capitalization', () => {
+        const editableSchemaMetadata = { editableSchemaFieldInfo: [] };
+        const filterText = 'TeSting DesCriptioN';
+        const { filteredRows, expandedRowsFromFilter } = filterSchemaRows(
+            rows,
+            editableSchemaMetadata,
+            filterText,
+            testEntityRegistry,
+        );
+
+        expect(filteredRows).toMatchObject([{ fieldPath: 'testing' }]);
+        expect(expandedRowsFromFilter).toMatchObject(new Set());
+    });
+
+    it('should properly filter schema rows based on editable description', () => {
+        const editableSchemaMetadata = {
+            editableSchemaFieldInfo: [
+                {
+                    fieldPath: 'customer',
+                    description: 'editable customer description',
+                    globalTags: null,
+                    glossaryTerms: null,
+                },
+            ],
+        };
+        const filterText = 'editable customer description';
+        const { filteredRows, expandedRowsFromFilter } = filterSchemaRows(
+            rows,
+            editableSchemaMetadata,
+            filterText,
+            testEntityRegistry,
+        );
+
+        expect(filteredRows).toMatchObject([{ fieldPath: 'customer' }]);
+        expect(expandedRowsFromFilter).toMatchObject(new Set());
+    });
+
+    it('should properly filter schema rows based on editable description regardless of capitalization', () => {
+        const editableSchemaMetadata = {
+            editableSchemaFieldInfo: [
+                {
+                    fieldPath: 'customer',
+                    description: 'EdiTable CuStoMer DesCriptioN',
+                    globalTags: null,
+                    glossaryTerms: null,
+                },
+            ],
+        };
+        const filterText = 'EdiTable CuStoMer DesCriptioN';
```

Comment:
isn't the point of this test to check that capitalization doesn't matter? I'd just make this all lowercase to actually test that (as opposed to the same piece of text that's in the description)